wagtail/images/views/images.py

```diff
@@ -58,11 +60,13 @@ def get_context_data(self, **kwargs):
 
         # Filter by collection
         self.current_collection = None
+        next_url = None
         collection_id = self.request.GET.get('collection_id')
         if collection_id:
             try:
                 self.current_collection = Collection.objects.get(id=collection_id)
                 images = images.filter(collection=self.current_collection)
+                next_url = reverse('wagtailimages:index') + "?" + urlencode({"collection_id": collection_id})
```

Comment:
Do we need this? Shouldn't we just use the current url, so we get tags (for instance) as well?

Reply:
Sure! I've updated it to use the current url so we don't lose information.